scripts/patterns-reorg/main.py

```diff
@@ -23,6 +23,8 @@
 from entries import TOP_LEVEL_ENTRIES
 from deleted_entries import DELETED_PAGES
 
+OLD_FOLDERS = ["start", "run", "verify", "transpile", "build"]
+REDIRECTS = (*TOP_LEVEL_ENTRIES, *DELETED_PAGES)
```

Comment:
I meant this:

```suggestion
REDIRECTS = determine_redirects((*TOP_LEVEL_ENTRIES, *DELETED_PAGES))
```

You'll need to update call sites